strawberryfields/engine.py

```diff
@@ -548,7 +548,7 @@ def run(self, program: Program, *, compile_options=None, **kwargs) -> Optional[R
                 if job.status == "failed":
                     message = (
                         "The remote job %s failed due to an internal "
-                        "server error. Please try again." % job.id
+                        "server error. Please try again. %s" % (job.id, job.meta)
```

Comment:
Any reason not to use new-style formatting here?

Reply:
Oh, I think this is an artifact of [Python's logger expecting `printf` string formatting](https://docs.python.org/3/library/logging.html#formatter-objects). At one point, this line must have been inside the `self.log.error()` function.

Now that it's been refactored out, it's best to update this to use new-style formatting.

Reply:
Done!